Common/cpp/Tools/RuntimeDecorator.cpp

```diff
@@ -113,6 +113,22 @@ void RuntimeDecorator::addNativeObjects(jsi::Runtime &rt,
   jsi::Value measureFunction = jsi::Function::createFromHostFunction(rt, jsi::PropNameID::forAscii(rt, "_measure"), 1, clb4);
   rt.global().setProperty(rt, "_measure", measureFunction);
   
+  auto runOnJS = [](
```

Comment:
do we need `runOnJS` to be defined here? I think it'd work better if it was defined in JS. This way user has to import it from reanimated, which allows us to:
1) avoid naming conflicts
2) eliminate lint warnings about method not being defined
3) allow us to provide TS typings
4) can also make it work when used from the JS thread (which may make sense because some worklets run both on JS and UI threads (I think we have it defined already)
5) will give us better flexibility on how we want to structure the way one can pass parameters

Reply:
Should it return a promise with the result of execution?
Or the execution will always return void for now? I'm thinking about TS types.

Reply:
@kmagiera  It is exported. I just don't want to make another conditional statement inside `toJSValue` method.
If you want to I can check there if the function name is equal to `runOnJS`.

Reply:
I think it still makes sense to have it in JS. There was a discussion whether we want this to return a function, or maybe pass arguments directly: `runOnJS(fun)(arg)` or `runOnJS(fun, arg)` – this would be easier for us to change if we had it in one place and not it two.